packages/service-library/src/servicelib/rabbitmq_errors.py

```diff
@@ -20,3 +20,13 @@ class RemoteMethodNotRegisteredError(BaseRPCError):
         "Could not find a remote method named: '{method_name}'. "
         "Message from remote server was returned: {incoming_message}. "
     )
+
+
+class GatheredRuntimeErrors(BaseRPCError):
```

Comment:
this is a pretty weird name. what is it for? from your code it looks like you put all the errors together. but what is the idea?